### DISCUSSION TITLE
I know nothing

### DISCUSSION THREAD

**Discussion by decourcy2** [WittleWolfie/WW-Blueprint-Core#78] [Q&A]
I am looking to mod in a new sword '3 Ring Broadsword/Temple Sword' or just make longsword a monk weapon. 
Would this framework mod help me do that? 
Thanks for your time.

**Comment by WittleWolfie** [comment]
It would. You don't strictly need it but I genuinely think it's the best way to get started modding, in particular because the tutorials are the most step-by-step and in-depth available. It also automatically handles a lot of little things that will cause problems, hopefully letting you just focus on the mechanical changes you need.
